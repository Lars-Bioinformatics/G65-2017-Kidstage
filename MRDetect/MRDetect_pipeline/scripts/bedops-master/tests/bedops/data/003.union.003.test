chr9	141127805	141127832	chr9:141127805-141127832,+,1 1 +
chrM	6	64	chrM:6-64,+,5 5 +
chrM	9	419	chrM:9-419,-,14117 17 -
chrM	11	422	chrM:11-422,-,3100 14 -
chrM	12	559	chrM:12-559,-,25584 17 -
chrM	13	64	chrM:13-64,+,13 6 +
chrM	34	255	chrM:34-255,+,53 7 +
chrM	67	213	chrM:67-213,+,103 9 +
chrM	69	153	chrM:69-153,+,16 7 +
chrM	183	213	chrM:183-213,+,14 6 +
