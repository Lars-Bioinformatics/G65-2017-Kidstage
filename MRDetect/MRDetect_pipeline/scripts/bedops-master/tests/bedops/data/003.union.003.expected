chr9	141127305	141128332	chr9:141127805-141127832,+,1 1 +
chrM	0	564	chrM:6-64,+,5 5 +
chrM	0	564	chrM:13-64,+,13 6 +
chrM	0	653	chrM:69-153,+,16 7 +
chrM	0	713	chrM:67-213,+,103 9 +
chrM	0	713	chrM:183-213,+,14 6 +
chrM	0	755	chrM:34-255,+,53 7 +
chrM	0	919	chrM:9-419,-,14117 17 -
chrM	0	922	chrM:11-422,-,3100 14 -
chrM	0	1059	chrM:12-559,-,25584 17 -
